lib/ui/components/RequestLogs/RequestLog.js

```diff
@@ -58,6 +63,13 @@ class RequestLog extends React.Component {
     return API.capturedRequests
       .filter((request) => includes(request.url, RequestLogState.query))
       .filter(filterLogByType)
+      .filter((request) => {
+          if(this.state.requestTimestamp != null) {
+            return request.startTime > this.state.requestTimestamp;
+          } else {
+            return request;
```

Comment:
the return type for filter is either true/false so if you want to keep the request you need to return true